docs/requirements.txt

```diff
@@ -3,7 +3,7 @@ flax
 funsor
 jax>=0.1.65
 jaxlib>=0.1.45
-nbsphinx
+nbsphinx==0.8.1
```

Comment:
`nbsphinx==0.8.2` does not render correctly (e.g. see `github_url` at [Bayesian regression tutorial](http://num.pyro.ai/en/stable/tutorials/bayesian_regression.html). I don't know how to fix the issue so would like to pin `nbsphinx` to a working version instead.